journal/src/main/scala/com/evolutiongaming/kafka/journal/ActionHeader.scala

```diff
@@ -96,25 +72,25 @@ object ActionHeader {
   final case class Append(
     range: SeqRange,
     origin: Option[Origin],
-    version: Option[Version],
+    version: Version = Version.obsolete,
```

Comment:
Technically, we do not need here and in 3 other places the default value, right?

All clients with versions since 0.0.152 have to provide these in payload over Kafka. Oldest client we have is 0.0.177, IIRC, thus all of them must provide `version` field, right?

I checked the history for those `journal/src/test/resources/com/evolutiongaming/kafka/journal` and related unit-tests - they are more than 6 years old, they were introduced in version 0.0.10 - I am leaning on upgrading them to state/format, which is expected at 0.0.177 - then we will require these fallbacks

Other option would be to introduce protocol version, but that could be overkill at this moment